lib/vauth/gsasl.c

```diff
@@ -38,81 +39,75 @@
 #include "curl_memory.h"
 #include "memdebug.h"
 
-CURLcode
-Curl_auth_create_scram_cf_message(struct Curl_easy *data,
-                                  const char *userp,
-                                  const char *passwdp,
-                                  struct gsasldata *gsasl,
-                                  char **outptr, size_t *outlen)
+bool Curl_auth_gsasl_is_supported(struct Curl_easy *data,
+                                  const char *mech,
+                                  struct gsasldata *gsasl)
 {
-  CURLcode result;
-  char *s;
-  size_t slen;
   int res;
 
   res = gsasl_init(&gsasl->ctx);
   if(res != GSASL_OK) {
-    infof(data, "GNU SASL init (%d):%s\n", res, gsasl_strerror(res));
-    return CURLE_REMOTE_ACCESS_DENIED;
+    infof(data, "GSASL init: %s\n", gsasl_strerror(res));
+    return false;
```

Comment:
We prefer `FALSE` - uppercase for both the boolean values when used.

Reply:
Thanks, fixed now.